internal/cmd/server/enable_rescue.go

```diff
@@ -43,7 +43,14 @@ var EnableRescueCmd = base.Cmd{
 			opts hcloud.ServerEnableRescueOpts
 		)
 		rescueType, _ := cmd.Flags().GetString("type")
+
 		opts.Type = hcloud.ServerRescueType(rescueType)
+		switch opts.Type {
+		case hcloud.ServerRescueTypeLinux64:
+			break
+		default:
+			return fmt.Errorf("invalid rescue type: %v", opts.Type)
```

Comment:
Could we have 1 extra case: 1 for unknown values, and 1 for deprecated values: the error message feels like a user error, but we dropped support for the rescue type.

Reply:
Implemented in 4de80c2